transform.js

```diff
@@ -58,8 +58,22 @@ function run(argv) {
   };
 
   const toSlug = (string = '', replacement = '-') => {
-    const words = string.match(WORD);
-    return (words || []).join(replacement);
+      const cyrillicToLatinMap = {
```

Comment:
Would you mind move this to the constant outside of this function to prevent its creation on every time `toSlug` is called?